apps/ledger-live-desktop/tests/specs/speculos/subAccount/sub.account.spec.ts

```diff
@@ -0,0 +1,35 @@
+import test from "../../../fixtures/common";
+import { Token } from "../../../enum/Tokens";
+import { specs } from "../../../utils/speculos";
+import { Application } from "tests/page";
+import { addTmsLink } from "tests/fixtures/common";
+
+const tokens: Token[] = [
+  Token.ETH_USDT,
+  Token.XLM_USCD,
+  Token.ALGO_USDT,
+  Token.TRON_USDT,
+  Token.BSC_BUSD,
+  Token.MATIC_DAI,
+];
+
+for (const [i, token] of tokens.entries()) {
+  test.describe.parallel("Sub Account @smoke", () => {
+    test.use({
+      userdata: "speculos-subAccount",
+      testName: `subAccount_${token.parentAccount.currency.uiName}`,
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${token.parentAccount.currency.uiName}] Sub Account`, async ({ page }) => {
+      addTmsLink(["B2CQA-1425"]);
```

Comment:
I don't get this one to be honest, aren't we suppose to add account through parent?

Reply:
Yes, but the issue is that there is multiple cases in 1 Ticket, I going to talk with manual QA to split the tiket: https://ledgerhq.atlassian.net/browse/B2CQA-640

Reply:
I just updated all the test names and linked them to the correct ticket according to @mcoursier-ledger.